cinema/serializers.py

```diff
@@ -1,6 +1,14 @@
 from rest_framework import serializers
-
-from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession
+from django.db import transaction
+from cinema.models import (
+    Genre,
+    Actor,
+    CinemaHall,
+    Movie,
+    MovieSession,
+    Order,
+    Ticket
+)
```

Comment:
Code Style issue: It's recommended to group the imports using parentheses if needed. Consider organizing your imports in a more readable way. For example, you can group them by their purpose or their source.